Lib/fontTools/merge.py

```diff
@@ -975,20 +1089,52 @@ def merge(self, fontfiles):
 
 		return mega
 
-	def _mergeGlyphOrders(self, glyphOrders):
+	def _mergeGlyphOrders(self, glyphOrders, cffTables):
 		"""Modifies passed-in glyphOrders to reflect new glyph names.
 		Returns glyphOrder for the merged font."""
 		# Simply append font index to the glyph name for now.
 		# TODO Even this simplistic numbering can result in conflicts.
 		# But then again, we have to improve this soon anyway.
+
+		# If we have CFF tables,
+		# rename topDictIndex charStrings
+		for n,cffTable in enumerate(cffTables):
+			td = cffTable.cff.topDictIndex[0]
+			d = {}
+			for i, (glyphName, v) in enumerate(td.CharStrings.charStrings.items()):
+				if glyphName not in ['.notdef']:
```

Comment:
wouldn't `if glyphName != ".notdef":` accomplish the same thing here?

Reply:
sure ;)